app/controllers/recipes_controller.rb

```diff
@@ -1,2 +1,39 @@
 class RecipesController < ApplicationController
+  def index
+    # @user = User.find(params[:user_id])
+    @recipes = Recipe.all
```

Comment:
`@recipes` variable should contain all recipes created by currently logged in user, which is `current_user`